paddle/fluid/operators/reader/open_files_op.cc

```diff
@@ -21,6 +21,22 @@ namespace reader {
 
 class MultipleReader : public framework::ReaderBase {
  public:
+  class ThreadBufferMap {
+   public:
+    std::vector<framework::LoDTensor>& operator[](
```

Comment:
A convention in Google C++ style guide is that if something is mutable, use the pointer; otherwise, use the const reference. It seems here the operator need to return type `std::vector<framework::LoDTensor>*`, other than `&`.

Or, please run `pre-commit install` and enable the new hook that calls `cpplint` for style checking.

Reply:
The `ThreadBufferMap` is designed to be a tiny thread-safe `map`. So I preferred to make its interfaces consistent with `std::map`. I'm not sure whether the convention you mentioned applies to functions' return values, I only find the following rule on function arguments:

'Parameters are either input to the function, output from the function, or both. Input parameters are usually values or const references, while output and input/output parameters will be pointers to non-const.'

I have enabled the `cpplint`, It didn't give any warning or error messages on the return value.